File: 0.C_CPP_JAVA Programs/JAVA/Program22.java
/*
2. Write Java program which accept number of rows and number of columns from user and display below pattern.
Input :  iRow = 4  iCol = 4
Output : A B C D
         a b c d
         A B C D
         a b c d
 */
import java.lang.*;
import java.util.*;

class DemoString
{
    void Pattern(int No1,int No2)
    {
        char ch1='A';
        char ch2='a';
        for (int i=0;i<No1;i++)
        {
            for(int j=0;j<No2;j++)
            {
                if(i%2==0)
                {
                    System.out.print("\t" + ch1);
                    ch1++;
                }
                else
                {
                    System.out.print("\t" + ch2);
                    ch2++;
                }

            }
            ch1='A';
            ch2='a';
            System.out.print("\n");
        }
    }
}
class Demo
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the Coloumn:");
        int No1=sobj.nextInt();

        System.out.println("Enter second row:");
        int No2=sobj.nextInt();

        DemoString dobj=new DemoString();

         dobj.Pattern(No1,No2);
    }
}

File: 1.Python/Program22.py
# 10. Write a program which accept number from user and return addition
# of digits in that number.
# Input : 5187934   Output : 37
def DigitAdd(No):
     iCnt=0
     iSum=0
     while No!=0:
       iRem=No%10
       iSum = iSum + iRem
       No=No//10
     return iSum
def Main():
   No=0
   iRet=0
   print("Enter the one number:")
   No=int(input())
   iRet=DigitAdd(No)
   print("Your Digit count is: ",iRet)
if __name__ == "__main__":
   Main()